docs/reference/helpers.rst

```diff
@@ -85,6 +85,20 @@ helper. The option expects either a string or an array of formats.
 
     {% media media, 'large' with {'srcset': ['small', 'big']} %}
 
+To render the image as ``<picture>`` element instead of ``<img>``, pass a ``picture`` key instead of ``srcset`` above:
```

Comment:
What should happen if both keys are provided? I think it should throw an exception.

Reply:
Do we already have one that‘s suitable?

Reply:
We can use a native one for this, since that exception is not meant to be caught. `\LogicException` would do IMO.